content/docs/user-guide/data-management/data-versioning.md

```diff
@@ -0,0 +1,43 @@
+# Data Versioning
+
+DVC enables [version control] for data science. But DVC does not actually
+implement versioning directly! Instead, DVC focuses on [codifying your data]:
+generating small [metafiles] that you can handle with standard [Git workflows]
+(commits, branching, pull requests, etc.).
+
+The resulting projects are neatly organized in the "space dimension", having
+only the files and directories needed at the time and without complicated, ad
+hoc file names like `2022-10-20_linear-model_v2-Carl`. Project versions live in
+the "time dimension" ([Git history]).
+
+![Versioned ML project](/img/versioned-project.png) _Navigate versions with Git
+commits_
+
+**Data version control** is the unifying trait across DVC features (data
+management and beyond).
+
+<admon icon="book">
+
+Refer to [Versioning Data and Models] to learn more.
+
+[versioning data and models]: /doc/use-cases/versioning-data-and-models
+
+</admon>
+
+[version control]:
+  https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control
+[codifying your data]: /doc/use-cases/versioning-data-and-models
+[metafiles]: /doc/user-guide/project-structure
+[git workflows]: https://www.atlassian.com/git/tutorials/comparing-workflows
+[git history]:
+  https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History
+
+<!--
+## Cloud versioning
+
```

Comment:
Just noticed something funny @iterative/websites : Commented-our headers still render in the right-side nav 😛 

See https://dvc-org-guide-data-mgmt-lfnbat.herokuapp.com/doc/user-guide/data-management/data-versioning:

![image](https://user-images.githubusercontent.com/1477535/197328554-579e4b5b-f798-4cdc-a363-ff1bcac52004.png)